### PR TITLE
Creating markdown file for decrement Exact

### LINKED ISSUE #898: [Term Entry] Math Methods: `.decrementExact()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.decrementExact()` Java method.

- In the `terms/` folder, make a new folder called `decrement-exact`
- Inside the `decrement-exact` folder, create a new markdown file called `decrement-exact.md`.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by isanketsaha**
### Description

Closes #898

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Dusch4593** [comment]
@isanketsaha First off, thank you so much for your interest in contributing to Codecademy Docs, especially during Hacktoberfest! 🚀

I'll be conducting the first review for this PR (we'll need at least two approvals ✅ before we merge). I'll tag you and let you know when I've finished.

> **Note:** Please make sure to check off the items near the top of this PR thread (under "Checklist"). 😄

File: content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md:31]:
```suggestion
// Main.java

public class Main {
```

File: content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md:24]:
```suggestion
The `num` parameter is of type `long` or `int`.
```

File: content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md:16]:
```suggestion
The **`Math.decrementExact()`** method returns the argument decremented by one. It throws an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows the specified [data type](https://www.codecademy.com/resources/docs/java/data-types) (`long` or `int`) of the argument.
```

File: content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/decrement-exact/decrement-exact.md:13]:
```suggestion
Title: '.decrementExact()'
Description: 'Returns the argument decremented by one, throwing an exception if the result overflows the datatype.'
Subjects:
- 'Computer Science'
  Tags:
- 'Functions'
- 'Methods'
- 'Arithmetic'
  CatalogContent:
- 'learn-java'
```

Empty lines aren't necessary in the entry meta-data.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@isanketsaha Finished wit my review and left you some comments/suggestions. Tag me once you're finished and I'll take another look. 😄

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:12]:
```suggestion
- 'Arithmetic'
  CatalogContent:
- 'learn-java'
```

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:27]:
```suggestion
The following example uses the `.decrementExact()` method to decrement the input by 1:
```

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:23]:
```suggestion
The `num` parameter is of type `long` or `int`.
```

We don't need a single list item here.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@isanketsaha Just a few more comments/suggestions. 😄

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (isanketsaha) [reply content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:23]: Done

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:6]:
```suggestion
Tags:
```

**Review by Dusch4593 — APPROVED** [review]
@isanketsaha LGTM! I had one more small point of feedback, but we are ready for the next review. 😄

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (yangc95) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:5]:
```suggestion
  - 'Computer Science'
```

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (yangc95) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:9]: Let's do a two-space indents for these bulleted tags here

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (yangc95) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:14]:
```suggestion
The **`Math.decrementExact()`** method returns the argument decremented by one. It throws an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows the specified [data type](https://www.codecademy.com/resources/docs/java/data-types) of the argument.
```

Comment (yangc95) [reply content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:14]: Since we mention the data type below, removed it from the intro

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (yangc95) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:29]:
```suggestion
// Main.java
```

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (yangc95) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:30]:
```suggestion
```

**Review by yangc95 — COMMENTED** [review]
@isanketsaha Did a second review! Please tag me once you've looked over the comments and let me know if you have any questions.

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (isanketsaha) [reply content/java/concepts/math-methods/terms/decrementExact/decrementExact.md]: @Dusch4593 , we need the line gap between the subject, and tags. else the content will overlap.

Screenshot 2022-10-21 at 4 22 29 PM
 

I am attaching [pictures.](URL)

Screenshot 2022-10-21 at 4 22 55 PM

Comment (isanketsaha) [reply content/java/concepts/math-methods/terms/decrementExact/decrementExact.md]: Let me know your thoughts.

**Comment by isanketsaha** [comment]
@yangc95 , ready for review.

**Review by yangc95 — APPROVED** [review]
LGTM!

@isanketsaha Prior to merge, would you be able to mark everything on the checklist above? 
Screen Shot 2022-10-21 at 10 51 01 AM

In addition, make sure to pass the checks with format scripts. We can walk you through this or assist you directly with them. Let us know how you'd like to proceed!

**Comment by Dusch4593** [comment]
@isanketsaha Can we also fill out the checklist in #1140?

cc: @yangc95

**Comment by isanketsaha** [comment]
@yangc95 please help me run the lint rules.

File: content/java/concepts/math-methods/terms/decrementExact/decrementExact.md
Comment (yangc95) [inline content/java/concepts/math-methods/terms/decrementExact/decrementExact.md:11]:
```suggestion
  - 'learn-java'
  - 'paths/computer-science'
```

**Comment by yangc95** [comment]
@isanketsaha live link: https://www.codecademy.com/resources/docs/java/math-methods/decrementExact

Screen Shot 2022-10-23 at 2 35 42 PM
You're currently anonymous; if you wish to become public, make sure to go onto your Codecademy profile and link your GitHub handle!

cc: @Dusch4593

**Pull request merged**
